src/Blocks/BlockOre.h

```diff
@@ -50,6 +50,11 @@ class cBlockOreHandler :
 				a_Pickups.push_back(cItem(E_ITEM_COAL));
 				break;
 			}
+			case E_BLOCK_QUARTZ_ORE:
+			{
+				a_Pickups.push_back(cItem(E_ITEM_QUARTZ));
+				break;
+		        }
```

Comment:
You're using a combination of spaces and tabs here. Could you change it to tabs?